atlp-rwanda/atlp-pulse-mobile
# Fixing sprint rating and comment model

### What does this PR do?

As a trainee, I should be able to see sprint ratings & Recent feedback. 

### Description of Task to be completed?

- [x] show only Phase with Rating
- [x] overlay need improving
- [x] show x-axis on stats graph

### How should this be manually tested?

- Clone repo
- Checkout branch `fx-sprint-ratings`
- Make sure that you set the environment variables referring to .env.example
- Run `npm install`, then `npm run dev` to check if the app is running successfully
- login using trainee credentials

### Screenshots

<img width="180" src="https://github.com/user-attachments/assets/7df6dcb8-56be-411e-a419-68a393721f7c" />
<img width="180" src="https://github.com/user-attachments/assets/046e805f-6d76-4d8c-90c0-38c610b8afe6" />
<img width="180" src="https://github.com/user-attachments/assets/7c2b1cb9-4dcf-4f35-bc47-1b021452f998" />
<img width="180" src="https://github.com/user-attachments/assets/1c52bf8f-002d-4062-8893-299928b4924d" />
